docs/onnx_custom_op/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# [cmake:onnx_custom_op]
+set(TARGET_NAME "onnx_custom_op")
+
+find_package(ngraph REQUIRED)
+find_package(InferenceEngine REQUIRED)
+
+add_library(${TARGET_NAME} STATIC onnx_custom_op.cpp)
+
+target_compile_options(${TARGET_NAME} PRIVATE -std=c++11)
+
+target_include_directories(${TARGET_NAME} PRIVATE ${InferenceEngine_INCLUDE_DIRS})
```

Comment:
don't have to add include directories manually since `IE::inference_engine` targets already contains this.